MecanumControllerInterface:
  ros__parameters:
    robot_frame_id: "base_link"
    odom_frame_id: "odom"
    robot_width: 0.3
    robot_length: 0.2
    wheel_radius: 0.06
    maximum_x_velocity: 1.0
    maximum_y_velocity: 1.0
    maximum_z_velocity: 1.0
